src/components/Card.js

```diff
@@ -16,6 +30,24 @@ class Card {
     });
   }
 
+  setIsLiked(isLiked) {
+    // set instance variable
+    this._isLiked = isLiked;
+    this._renderLikes();
+  }
+
+  getLikes() {
+    return this._isLiked;
+  }
+
+  _renderLikes() {
+    if (this._isLiked) {
+      this._likeButton.classList.add("card__like-button_active");
+    } else {
+      this._likeButton.classList.remove("card__like-button_active");
+    }
+  }
+
```

Comment:
Needs correction: ` this._handleLikeClick` (line 14) should be used as the listener of the like button. `_handleLikeIcon` should be removed